### DISCUSSION TITLE
How to limit Province to just one?

### DISCUSSION THREAD

**Discussion by orschiro** [woocommerce/woocommerce#50750] [Checkout Flow]
Hi team, 

We are based in Spain and only want to ship to the province "Valencia".

By default, the customer can choose from all provinces in Spain.

How can I limit this selection to just provide "Valencia"?

Thank you!

![image](https://github.com/user-attachments/assets/dc98511f-a2c7-4174-be77-0a33b178b216)

![image](https://github.com/user-attachments/assets/429e0779-d954-40fb-89e7-ac726c42874b)

**Comment by moon0326** [comment]
Hi there!

I think you're looking for  `woocommerce_states` filter. Please take a look at [Add/Modify States](https://woocommerce.com/document/addmodify-states/)  documentation.

**Reply by orschiro** [reply]
Thanks! :)

It worked with this code:

```php
add_filter( 'woocommerce_states', 'tl45r_custom_woocommerce_states' );

function tl45r_custom_woocommerce_states( $states ) {

    // Only show specific provinces (Valencia - V) for Spain (ES)
    $states['ES'] = array(
        'V' => __( 'Valencia', 'woocommerce' ),
    );

    return $states;
}
```
